pennylane/ops/op_math/pow_class.py

```diff
@@ -237,3 +242,21 @@ def generator(self):
         See also :func:`~.generator`
         """
         return self.z * self.base.generator()
+
+    @property
+    def arithmetic_depth(self) -> int:
+        return 1 + self.base.arithmetic_depth
+
+    def simplify(self, depth=-1) -> Union["Pow", Identity]:
+        if depth == 0:
+            return self
+        if self.z == 0:
+            return Identity(wires=self.wires[0])
+        if isinstance(self.base, ControlledOp):  # Pow(Controlled(base)) = Controlled(Pow(base))
+            return ControlledOp(
+                base=Pow(self.base.base, z=self.z),
```

Comment:
This example brings up a good question, suppose a certain simplification doesn't change the `depth`, (eg. a*(b+c) --> a*b + a*c). In this case, how can we make it clear to the user that simplification has occurred? Is there a way to quantify these steps so that we can give the user control over how many of these calls to make?

Reply:
We removed the `depth` argument from the `simplify` method. We can add a `max_recursion` argument if we see that simplifying becomes computationally expensive.

Reply:
And yes @albi3ro ! I forgot to simplify `self.base.base`.